janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphIndexTest.java

```diff
@@ -1124,6 +1126,7 @@ public void testGraphCentricQueryProfiling() {
         if (indexFeatures.supportNotQueryNormalForm()) {
             newTx();
             assertEquals(3, tx.traversal().V().or(__.has("name", "bob"), __.has("prop", "val")).count().next());
+            assertEquals(3, tx.traversal().V().or(__.has("name", "bob"), __.has("prop", "val")).toList().size());
```

Comment:
What is the reason to change these queries from `count().next()` to `toList().size()`? I see that the result should be equivalent, but therefore I don't see the necessity to change the query.

Reply:
Now that count() queries are optimized by `JanusGraphMixedIndexCountStrategy`, I just add this extra check to ensure correctness when JanusGraphMixedIndexCountStrategy does not apply.